### PR TITLE
Dev mac

### PR DESCRIPTION
**Pull request by GAIGResearch**
Compatibility with Mac. Log of changes here: https://docs.google.com/document/d/1Tzn1owFvdbBmFoNaDwqYiG8bbjD-YkTNLAC-DVr_CQE/edit?usp=sharing

### CODE REVIEW

File: GUI/include/MapLayer.h [hunk 1]
```diff
@@ -19,13 +19,13 @@ class MapLayer : public RenderLayer<GameState>
 			for (int x = 0; x < board.getWidth(); ++x)
 			{
 				auto& targetTile = board.getTile(x, y);
-				sf::Texture& texture = assetCache.getTexture("tile_" + std::to_string(targetTile.tileTypeID));
+				sf::Texture& texture = RenderLayer<GameState>::assetCache.getTexture("tile_" + std::to_string(targetTile.tileTypeID));
```
Comment (CommanderCero) [inline GUI/include/MapLayer.h:22]: why is adding RenderLayer<GameState>:: necessary?
I mean we only inherit from one class

**Review by CommanderCero — CHANGES_REQUESTED** [review]

File: Stratega/include/Representation/BuildContext.h [hunk 2]
```diff
@@ -1,9 +1,10 @@
 #pragma once
 #include "Recast.h"
+#include <cstdint>
 
 namespace SGA
 {
-	typedef __int64 TimeVal;
+	typedef __int64_t TimeVal;
```
Comment (CommanderCero) [inline Stratega/include/Representation/BuildContext.h:7]: shouldnt it be int64_t instead of __int64_t

**Review by CommanderCero — CHANGES_REQUESTED** [review]

File: GUI/include/MapLayer.h (hunk 1 above)
Comment (GAIGResearch) [reply GUI/include/MapLayer.h:22]: Done.

File: Stratega/include/Representation/BuildContext.h (hunk 2 above)
Comment (GAIGResearch) [reply Stratega/include/Representation/BuildContext.h]: Changed to int64_t

**Pull request merged**